lib/src/code_field/text_editing_value.dart

```diff
@@ -94,6 +94,14 @@ extension TextEditingValueExtension on TextEditingValue {
     );
   }
 
+  TextEditingValue typed(String text) {
```

Comment:
This method used only in tests. May be create private extension on `TextEditingValue` there?

Reply:
`+`